pages/reports/index.vue

```diff
@@ -0,0 +1,269 @@
+    };
+
+    if (organizationsType.acronym === 'GOV') {
+      const gorverment = countries.find((o) => o.code === record.organization.government); // eslint-disable-line no-loop-func
+
+      organizationInfo = {
+        organizationId: gorverment.code,
+        name          : gorverment.name,
+      };
+    }
+
+    let organization = section.organizations.find((o) => o.organizationId === organizationInfo.organizationId);
+
+    if (!organization) {
+      organization = { ...organizationInfo, participants: [] };
+      section.organizations.push(organization);
+    }
+
+    // ^^^ lookup organizations ^^^
+
+    // vvv Add participants vvv
+
+    const participant = { ...record };
+
+    if (participant.country) participant.countryName = (countries.find((o) => o.code === participant.country) || {}).name;
+
+    organization.participants.push(participant);
+
+    // ^^^ Add participants ^^^
+
+    // if (cursor.skip > 250) break; // Test for 250 records only
+  }
+
+  // vvv SORT  vvv
+
+  reportData.sections = _.sortBy(reportData.sections, 'position');
+  reportData.sections.forEach((section) => {
+    section.organizations = _.sortBy(section.organizations, (o) => `${o.name}`.toLowerCase()); // eslint-disable-line no-param-reassign
+    section.organizations.forEach((organization) => {
+      // eslint-disable-next-line no-param-reassign
+      organization.participants = _.sortBy(organization.participants, (o) => `${o.firstName} ${o.lastName}`.toLowerCase());
+      // Participants must be sorted using reg position
+    });
+  });
+
+  return reportData;
+}
+
+function isParty(government) {
+  if (government === 'us') return false; // TODO
+  if (government === 'va') return false; // TODO
+
+  return !!government;
+}
+
+function createApiAuthenticatedHttpClient(token) {
+  return ky.create({
+    prefixUrl: process.env.NUXT_ENV_KRONOS_API,
+    hooks    : { beforeRequest: [ (request) => { request.headers.set('Authorization', `Token ${token}`); } ] },
+  });
+}
+
+function createHandlebarsInstance() {
+  const hb = Handlebars.create(); // Creates an isolated Handlebars environment to avoid global registrations
+
+  hb.registerHelper('date', () => {
+    const d    = new Date();
+    let month  = `${d.getMonth() + 1}`;
+    let day    = `${d.getDate()}`;
+    const year = d.getFullYear();
+
+    if (month.length < 2) month = `0${month}`;
+    if (day.length < 2) day = `0${day}`;
+
+    return [ year, month, day ].join('-');
+  });
+
+  hb.registerHelper('dateFormat', (value) => toDateFormat(value, 'dd LLLL yyyy'));
+
+  hb.registerHelper('dateRange', (startDate, endDate) => {
+    startDate = new Date(startDate); // eslint-disable-line no-param-reassign
+    endDate   = new Date(endDate); // eslint-disable-line no-param-reassign
+    if (startDate.getMonth() === endDate.getMonth()) {
+      return `${toDateFormat(startDate, 'dd')}-${toDateFormat(endDate, 'dd')} ${toDateFormat(endDate, 'LLLL yyyy')}`;
```

Comment:
what not groups the enddate for into one?

```suggestion
      return `${toDateFormat(startDate, 'dd')}-${toDateFormat(endDate, 'dd LLLL yyyy')}`;
```

Reply:
Done